google/cloud/bigtable/CMakeLists.txt

```diff
@@ -211,6 +211,8 @@ create_bazel_config(google_cloud_cpp_bigtable)
 if (BUILD_TESTING)
     add_library(
         bigtable_client_testing # cmake-format: sort
+        admin/mocks/mock_bigtable_instance_admin_connection.h
```

Comment:
Maybe in a separate PR, but I think we would want a separate library for public mocks (as opposed to helpers for our own tests / mocks for stubs, etc.)

Reply:
Yeah you're right. I'll add `google_cloud_cpp_bigtable_mocks` in a future PR.